src/components/Section/Section.js

```diff
@@ -0,0 +1,55 @@
+import React, {
+  memo,
+  useState,
+} from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  Checkbox,
+  Headline,
+} from '@folio/stripes/components';
+
+import css from './Section.css';
+
+export const Section = memo(({
+  headerCaption,
+  isOptional,
+  id,
+  children,
+}) => {
+  const [isChecked, setChecked] = useState(true);
+
+  const headline = (
+    <Headline
+      size="1.17rem"
```

Comment:
Please use one of the available size values `size: PropTypes.oneOf(['small', 'medium', 'large', 'x-large', 'xx-large'])`

Reply:
Can I add a class with font-size property to this component? The thing is I'd like components to be similar in style (RecordTypesSelect has a headline with 1.17rem font-size value). If it doesn't matter I can replace a size value with a large value.

Reply:
For now, just use one of the values provided by `Headline` component. I will check the styles in `RecordTypesSelect` component.